CHANGELOG.md

```diff
@@ -6,6 +6,45 @@
 
 ### Fixes
 
+## v1.32.0 (2023-08-14)
+
+### Enhancements
+
+- Add text label to copy buttons on tags ([@Yushmanth-reddy](https://github.com/Yushmanth-reddy) in [#1547](https://github.com/jaegertracing/jaeger-ui/pull/1547))
+
+### Fixes
+
+- Remove unused redux-async-middleware package ([@anshgoyalevil](https://github.com/anshgoyalevil) in [#1674](https://github.com/jaegertracing/jaeger-ui/pull/1674))
+- Remove unused source-map-explorer package ([@anshgoyalevil](https://github.com/anshgoyalevil) in [#1671](https://github.com/jaegertracing/jaeger-ui/pull/1671))
+- Remove unused d3-scale package ([@anshgoyalevil](https://github.com/anshgoyalevil) in [#1670](https://github.com/jaegertracing/jaeger-ui/pull/1670))
+- Remove unused bluebird package ([@yurishkuro](https://github.com/yurishkuro) in [#1669](https://github.com/jaegertracing/jaeger-ui/pull/1669))
+- Migrate ScoreCard.test.js from Enzyme to React Shallow Renderer ([@anshgoyalevil](https://github.com/anshgoyalevil) in [#1653](https://github.com/jaegertracing/jaeger-ui/pull/1653))
+- Fix the tooltip position of copy value and JSON Buttton ([@anikdhabal](https://github.com/anikdhabal) in [#1650](https://github.com/jaegertracing/jaeger-ui/pull/1650))
+- Bump @types/object-hash from 3.0.2 to 3.0.3 ([@dependabot](https://github.com/dependabot) in [#1658](https://github.com/jaegertracing/jaeger-ui/pull/1658))
```

Comment:
We could probably remove the version bumps. I don't think it provides much value for users, and drowns out the community contributions.

Reply:
The 1.31.0 has dependabot PRs in the changelog. 

cc) @yurishkuro

Reply:
https://github.com/jaegertracing/jaeger-ui/pull/1544

I have removed it for now. It would be good to be consistent